.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "4.0.0"
```

Comment:
I'm not super keen on how we've had to bump node, but 0.10 is really old. if we have to do this, then at least this should be "4"

Reply:
Yeah, true, things seemed to already be wired up fine for FF.  But, it wasn't until I bumped the node version that it all completed just fine.  Are you saying that the version in the travis yaml should be `4` instead of the specific `4.0.0` I put in there?

Reply:
yes, `"4"`

do that and we'll call it good. 

the only reason I've been reluctant to bump node versions is b/c that in my mind is a breaking change - and I hadn't even planned to cut releases for these PRs that only affect README and docs site. I mean, I think that's what happened to us. someone upstream decided that they don't support 0.1 (and who can blame them), but they prob just did a patch or minor bump on their own package and now we're stuck.

That said we prob shouldn't care about this. it would only affect people running `npm i` on node < 4. Let's wait for someone to complain about that.

Or maybe the deal w/ node is you can safely drop support for older versions when it reaches end-of-life w/o worrying about who you screw over down stream.

I suggest this: 
- add an `engines` entry to package.json indicating >4: https://docs.npmjs.com/files/package.json#engines
- add entries to the Upcoming Changes section of the CHANGELOG (should do that for README change anyway)

That way, if anyone does comma complain' 6 mo down the road we've left enough breadcrumbs for ourselves and them to maybe figure out WTF happened.

Reply:
I wonder if we ought to have travis run in 4, 6, 8 while we're at it? - might as well, it's not llike we'll be running tests that often.

Reply:
Thanks, all great suggestions.  I'll do multiple versions of node for Travis (4, 6, 8) add an engines entry, and update the CHANGELOG.  Then I'll make another comment when all those commits are pushed.